Example/HorizonCalendarExample/HorizonCalendarExample.xcodeproj/project.pbxproj

```diff
@@ -366,7 +366,7 @@
 			buildSettings = {
 				ASSETCATALOG_COMPILER_APPICON_NAME = AppIcon;
 				CODE_SIGN_STYLE = Automatic;
-				DEVELOPMENT_TEAM = 5Q5SGQT2R4;
+				DEVELOPMENT_TEAM = C835JB7RUE;
```

Comment:
new phone, lost my contacts. leave a comment with your number below

Reply:
I laughed so hard at this.